source/docs/advanced/graphql/dataloaders-and-cache-invalidation.md

```diff
@@ -79,6 +79,10 @@ The second one is more specific to GraphQL. To understand its gain, we should co
 
 If the product `PANT-01` was an upsell of all other products in the `pants` category, the inventory API would be requested again when resolving the `upsells.qty` field. Query-level caching prevents those extra-calls, by reusing the response from `products.qty` that has already been fetched previously to resolve data.
 
+### Clearing the cache
+
+Clearing the cache depends on which magento is used. For Magento1 please refer to [Magento 1 advanced section](/docs/magento1/advanced.html#Clearing-Front-Commerce-cache), and for Magento2 please refer to [Magento 2 advanced section](/docs/magento2/advanced.html#Clearing-Front-Commerce-cache)
```

Comment:
**nitpick:** it would be something that each platform might end up having and the sentence is a tad specific I think

**suggestion:** use a list for the related pages

```suggestion
Clearing the cache manually is possible on some of the supported platforms. Read dedicated pages for details:
- [Magento 1](/docs/magento1/advanced.html#Clearing-Front-Commerce-cache)
- [Magento 2](/docs/magento2/advanced.html#Clearing-Front-Commerce-cache)
```